Runtime/Scripts/Context.cs

```diff
@@ -151,6 +151,11 @@ public void DeleteMediaStreamTrack(IntPtr track)
             NativeMethods.ContextDeleteMediaStreamTrack(self, track);
         }
 
+        public IntPtr CreateVideoRenderer()
+        {
+            return NativeMethods.CreateVideoRenderer(self);
+        }
+
```

Comment:
Please add the method for disposing the video renderer

Reply:
ok, I added this commit https://github.com/Unity-Technologies/com.unity.webrtc/pull/181/commits/0892afb05812d4f3aec2e678d81ac64610f3984a